src/main/java/de/retest/recheck/report/TestReportFilter.java

```diff
@@ -96,16 +97,23 @@ static StateDifference filter( final StateDifference stateDifference, final Filt
 		AttributesDifference attributesDifference = elementDiff.getAttributesDifference();
 		LeafDifference identifyingAttributesDifference = elementDiff.getIdentifyingAttributesDifference();
 		Collection<ElementDifference> childDifferences = elementDiff.getChildDifferences();
+
 		if ( elementDiff.hasAttributesDifferences() ) {
 			attributesDifference = filter( elementDiff.getElement(), elementDiff.getAttributesDifference(), filter );
 		}
+
 		if ( elementDiff.hasIdentAttributesDifferences() ) {
```

Comment:
Does it make sense, that we move this into a separate method:
```
LeafDifference identifyingAttributesDifference = filterIdentifyingAttributes(elementDiff, filter)
```

Reply:
IMHO it is OK to stick to the given style.